test/index.js

```diff
@@ -0,0 +1,40 @@
+var test = require('tape'),
```

Comment:
test here still using Tape. Make sure to update to Mocha + Chai, and refactor readability of tests